test/integration/text-to-speech.test.js

```diff
@@ -53,7 +54,14 @@ describe('text to speech_integration', () => {
       });
 
       synthStream.on('close', (code, reason) => {
-        done();
+        console.log(code);
+        if (code === 1000) {
+          done();
+        } else if (code === undefined) {
+          // Do nothing: this is in order to prevent done() being called twice
```

Comment:
I see the comment inline, but can you elaborate more on this? It seems that normally if `code === undefined` we would go to the `else` block and call `fail();`. Do we not want to fail the test if it is `undefined`?

Reply:
Normally yes but this undefined is actually a duplicate call which creates a duplicate `done` call which is no longer valid

Reply:
Why is there a duplicate call on the `close` event?

Reply:
Your guess is as good as mine but I did test the current version and this has already existed and only came up now due to updating Jest

Reply:
Hmm ok